roles/config-chrony/tasks/prereq.yml

```diff
@@ -4,24 +4,21 @@
   package:
     name: '{{ item }}'
     state: installed
-  with_items:
-  - chrony
-  - firewalld
-  - python-firewall
+  loop: "{{ config_chrony_packages }}"
```

Comment:
The `package` module actually takes a list, so just pass the `config_chrony_packages` list to the `name` parameter should do.

Reply:
Fixed and pushed.